packages/slate-editor/src/components/LoadingPlaceholder/ImageWithLoadingPlaceholder.tsx

```diff
@@ -73,12 +73,12 @@ export const ImageWithLoadingPlaceholder = forwardRef<HTMLDivElement, Props>((pr
             className={classNames(styles.ImageWithLoadingPlaceholder, attributes.className)}
             style={{
                 aspectRatio: aspectRatio ? String(aspectRatio) : undefined,
-                backgroundImage: url ? `url("${url}")` : undefined,
+                backgroundImage: isLoaded ? `url("${src}")` : undefined,
                 ...attributes.style,
             }}
             ref={ref}
         >
-            {url && <img className={styles.Image} src={url} alt={alt} />}
+            {isLoaded && <img className={styles.Image} src={src} alt={alt} />}
```

Comment:
This is the fix. Instead of using the "blob" `url` returned from `useImage`, we rely on the original `src` string. While `useImage` is still used for progress reporting.